### DISCUSSION TITLE
What can I do when I don't see emmc on T95Z Plus?

### DISCUSSION THREAD

**Discussion by WizzardSK** [ophub/amlogic-s9xxx-armbian#2667] [Amlogic]
I flashed the latest Armbian for T95Z Plus on microsd, but when I boot it from microsd, there is no emmc visible in lsblk command.
How can I install it to emmc?
https://github.com/ophub/amlogic-s9xxx-armbian/releases/download/Armbian_noble_save_2024.11/Armbian_24.11.0_amlogic_s912-t95z-plus_noble_6.6.62_server_2024.11.20.img.gz

**Comment by ophub** [comment]
https://github.com/ophub/amlogic-s9xxx-armbian/tree/main/documents#1216-how-to-resolve-the-issue-of-io-errors-while-writing-to-emmc
